modules/services/mako.nix

```diff
@@ -302,6 +313,15 @@ in {
     assertions =
       [ (hm.assertions.assertPlatform "services.mako" pkgs platforms.linux) ];
 
+    xdg.dataFile."dbus-1/services/fr.emersion.mako.service".text =
+      generators.toINI { } {
+        "D-BUS Service" = {
+          Name = "org.freedesktop.Notifications";
+          Exec = getExe cfg.package;
+          SystemdService = "mako.service";
+        };
+      };
```

Comment:
I saw [in `dunst`](https://github.com/nix-community/home-manager/blob/5408e27961599b1350b651f88715daf6e67244a7/modules/services/dunst.nix#L135) we just reuse the dbus service file provided from the repo. 
I wanted to use [the dbus service file provided by `mako`](https://github.com/emersion/mako/blob/a69041b0fde6ea7808c0e883c4b82b39da40641e/fr.emersion.mako.service.in), but unlike the one provided by `dunst` it lacks the `SystemdService` field. The [arch guys seem to patch this file](https://github.com/archlinux/svntogit-community/blob/2fae8c15a0459ac6562daee5b0d39ac6f038ba13/trunk/systemd-dbus-activation.patch) to add the `SystemdService` field. I am going to see what I can do upstream so we can avoid writing this our self.

Reply:
It seems like the `SystmedService` line was purposefully removed, https://github.com/emersion/mako/commit/dbd9c2bc98ef0cec7801a049f4bf8d61f7be36d0, hoping to find out why.

Also I opened a patch to nixpkgs https://github.com/NixOS/nixpkgs/pull/191448 copying arch.